pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail_test.go

```diff
@@ -0,0 +1,78 @@
+package jobrunaggregatoranalyzer
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	fet "github.com/glycerine/golang-fisher-exact"
+)
+
+func TestRequiredPassCount(t *testing.T) {
+	for numberOfPayloadAttempts := range requiredPassesByPassPercentageByNumberOfAttempts {
+		requiredPassCounts := []int{}
+		previousRequiredPassCount := numberOfPayloadAttempts
+		for corpusPassPercentage := 100; corpusPassPercentage >= 0; corpusPassPercentage-- {
+			requiredPassCount := 0
+			for possiblePassCount := previousRequiredPassCount; possiblePassCount >= 0; possiblePassCount-- {
+				corpusPassCount := corpusPassPercentage * 7 // assume 400 runs in a week
+				corpusFailCount := 700 - corpusPassCount    // assume 400 runs in a week
+				payloadFailCount := numberOfPayloadAttempts - possiblePassCount
+
+				_, leftp, _, _ := fet.FisherExactTest(corpusPassCount, possiblePassCount, corpusFailCount, payloadFailCount)
```

Comment:
```
[ 11 12 ]
[ 21 22 ]
```

from https://github.com/openshift/ci-tools/blob/7f9fe21d7590d1fff373736ff6dd805edee0f2fd/pkg/jobrunaggregator/jobrunaggregatoranalyzer/required-pass-rate.py#L32-L33

@stevekuznetsov you sure?